battleship.py

```diff
@@ -154,7 +173,21 @@ def isVertical(ship):
 Returns: bool
 '''
 def isHorizontal(ship):
-    return
+    y = []
+    for row in range(len(ship)):
+        if ship[0][0] == ship[row][0]:
+    
+            y.append(ship[row][1])
+        else :
+            return False
+    y = sorted(y)
+    for i in range(1,len(y)):
+        if (y[i] == y[i-1]+1):
+            continue
+        else :
+            return False

```

Comment:
Use not equal to avoid using continue in the if statement

Reply:
Also, resolve conflicts by deleting pycache and then clicking on the resolve conflicts button for battleship.py